verification/src/http_server/handlers/verification/solidity/standard_json.rs

```diff
@@ -31,7 +34,7 @@ pub async fn verify(
         creation_tx_input: &params.creation_bytecode,
         deployed_bytecode: &params.deployed_bytecode,
     };
-    compile_and_verify_handler(&compilers, input, false)
-        .await
-        .map(Json)
+    let response = compile_and_verify_handler(&compilers, input, true).await?;
```

Comment:
Why it became `true` while being `false` before?

Reply:
oops, copypaste